nopassword/forms.py

```diff
@@ -59,6 +55,9 @@ def clean_username(self):
         return username
 
     def save(self, request, login_code_url='login_code', domain_override=None, extra_context=None):
+        if 'user' not in self.cleaned_data:
+            return ""
```

Comment:
Does it make sense to return an empty string in this case? Shouldn't we just raise a generic exception or something like that?

Reply:
The idea is that it fails silently. Do you it'd be better to raise an exception here and catch it in the view, even though it'll do nothing and continue with the same flow?

Reply:
Yeah I think thats a bit nicer. Our response has more to do with the View and not with the Form per se I guess?

Reply:
Ok, makes sense.
I'll do the changes. Thanks!

Reply:
@larsjeh I did the changes, but I created a new type of Exception for this. If we use the generic one we'd be masking any other exception that may arise. This way I think we'll only ignore this particular exceptions and break the flow for any other exceptions and be alerted when they happen.
What do you think?